src/realm/object-store/shared_realm.hpp

```diff
@@ -357,7 +357,6 @@ class Realm : public std::enable_shared_from_this<Realm> {
     bool compact();
     // For synchronized realms, the file written will have the client file ident removed.
     void write_copy(StringData path, BinaryData encryption_key);
-    void write_copy_without_client_file_id(StringData path, BinaryData encryption_key, bool allow_overwrite = false);
```

Comment:
Would we need the `allow_overwrite` somewhere and should keep it in the new `write_copy` as well?

Reply:
`allow_overwrite` was something I added for my own convenience at some point,  so not something that is required. It would require some more changes down the line if it should be supported for the local realm case, so I decided to drop it.

Reply:
Gotcha! Thanks for the explanation. 👍